src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -124,7 +126,7 @@
 
 (defn- share-qr-code
   [{:keys [share-qr-type qr-image-uri component-width customization-color full-name
-           profile-picture emoji on-share-press]
+           profile-picture emoji on-share-press watched-account?]
```

Comment:
Based on component Figma props, we should use a `type` prop which could have different values (`profile`, `wallet`, `saved-address` and `watched-address`) instead of a flag.

Reply:
I believe it was implemented this way to make it clearer since there is no different in the `saved-address` and `wallet`. Before, we decided not to follow the exact prop structure @J-Son89

Reply:
Agree with @briansztamfater 

I think this shouldn't be a flag, instead we should mirror figma props as much as possible (as we agreed).

If the change is minimal, then it should also  be easy to handle internally when the received type is `watched-account`.

Reply:
+1 on what @briansztamfater said.